java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -2550,6 +2552,56 @@ public T touch(@NonNull String[] keys) {
         return getThis();
     }
 
+    /**
+     * Counts the number of set bits (population counting) in a string of specified <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/bitcount/">redis.io</a> for details.
+     * @param key The key to count set bits of.
+     * @return The number set bits in the string.
```

Comment:
Apply below too
```suggestion
     * @return Command Response - The number set bits in the string.
```

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00